modules/metrics/prometheus/src/main/scala/higherkindness/mu/rpc/prometheus/PrometheusMetrics.scala

```diff
@@ -54,6 +54,12 @@ import io.prometheus.client._
  *    - "unknown-status"
  *    - "unreachable-error"
  *
+ * The `buildFullTotal` method contains the following metrics:
```

Comment:
Maybe we can name it as `buildAllTotalMetrics`, would it make sense?

Reply:
I think "full" is more appropriated, although potentially improved. I'm referring here to a more complete metrics than something that including more metrics. wdyt?